### DISCUSSION TITLE
Can't get gh cli ssh connections to work, doing something wrong?

### DISCUSSION THREAD

**Discussion by Req** [community/community#53588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Trying to clone a repo from an organization and it's failing due to access rights, but `gh auth status` says I am logged in though!

**Clone**
```
$ gh repo clone xxx/xxx
Cloning into 'xxx...
warning: templates not found /usr/share/git-core/templates
The authenticity of host 'github.com (140.82.121.3)' can't be established.
ECDSA key fingerprint is SHA256:p2QAMXNIC1TJYWeIOttrVc98/R1BUFWu3/LiyKgUfQM.
Are you sure you want to continue connecting (yes/no)? yes
Failed to add the host to the list of known hosts (/home/joel/.ssh/known_hosts).
Load key "/home/joel/.ssh/id_rsa": Permission denied
Load key "/home/joel/.ssh/id_ed25519": Permission denied
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
exit status 128
```

**Auth status**
```
$ gh auth status
github.com
  ✓ Logged in to github.com as Req (/home/joel/snap/gh/502/.config/gh/hosts.yml)
  ✓ Git operations for github.com configured to use ssh protocol.
  ✓ Token: *******************
```

Normal `git clone ...` with SSH and ` ssh -T git@github.com` works fine.

Possibly related, I'm also unable to add extensions because of the same public key issue. For example `$ gh extension install matt-bartel/gh-clone-org` fails.

**Comment by myselfhimself** [comment, marked as answer]
Hello,

if you installed `gh` via snap, [try this fix](https://forum.snapcraft.io/t/feature-request-post-install-messages/33273)

(or in short: run `snap connect gh:ssh-keys` to allow the snap-contained `gh` executable to access ssh keys).
